### DISCUSSION TITLE
请问提交 pr 的 user-name 是 github 的 username 还是 openbuild 的 username

### DISCUSSION THREAD

**Discussion by liyincode** [openbuildxyz/Web3-Frontend-Bootcamp#271] [Q&A] [answered]
<img width="802" alt="image" src="https://github.com/openbuildxyz/Web3-Frontend-Bootcamp/assets/25540709/101951e1-05cb-46c1-ad90-1dc367c5fc0e">

请问提交 pr 的 user-name 是 github 的 username 还是 openbuild 的 username

**Comment by linghuccc** [comment, marked as answer]
"每节课都有相应的实战任务，需要在 members/[github_id] 文件夹下按照 task[n] 的命名方式创建任务文件夹，如：task1、task2。"

“注册的 PR，命名为 init: user-name，例如 init: Beavnvvv”

前面写的是 github ID, 这里用 user-name 确实有点 misleading，从给的范例来看，老师的意思应该是 github ID, 因为 Beavnvvv 就是 github ID（对应的 github user-name 是 Yooao）

**Reply by liyincode** [reply]
好的，感谢
